### PR TITLE
Shorten module names

### PR DESCRIPTION
**Pull request by wingo**
See https://github.com/SnabbCo/snabbswitch/pull/742.

### CODE REVIEW

File: src/core/main.lua [hunk 1]
```diff
@@ -33,37 +33,34 @@ function main ()
...
-   end
-   program = select_program(program, args)
+   local program, args = select_program(parse_command_line())
    if not lib.have_module(modulename(program)) then
       print("unsupported program: "..program:gsub("_", "-"))
-      print()
-      print("Rename this executable (cp, mv, ln) to choose a supported program:")
-      print("  snabb "..(require("programs_inc"):gsub("\n", " ")))
-      os.exit(1)
+      usage(1)
```
Comment (kbara) [inline src/core/main.lua]: Is upstream ok with this?

**Comment by kbara** [comment]
The concept is great. The tests need to pass.

File: src/core/main.lua (hunk 1 above)
Comment (wingo) [inline src/core/main.lua]: Ahem :)  See SnabbCo#742 :)

Comment (wingo) [inline src/core/main.lua]: To be more clear, I don't know yet.  However I think removing the redundancy in the names does appear to be a goal; I think if we had named the binary snabb_lwaftr, some upstream code would have already mapped it to program.lwaftr.  So, we'll see how the upstream PR goes.  If there are additional changes to be made there we can merge them in.  I suggest we merge this PR now so as to already have renamed our module tree.

**Comment by kbara** [comment]
If you have reason to believe upstream will accept the changes to core files, LGTM.

**Comment by wingo** [comment]
Yeah I think upstream will accept this somehow, and the desire to rename the modules now is winning the tradeoff in my mind.  I merge.

**Pull request merged**